ftest.sh

```diff
@@ -111,6 +111,7 @@ done' EXIT
 DAOS_BASE=${SL_PREFIX%/install}
 if ! pdsh -l jenkins -R ssh -S -w "$(IFS=','; echo "${nodes[*]}")" "set -ex
 ulimit -c unlimited
+current_host=$(whoami)
```

Comment:
```suggestion
current_username=$(whoami)
```

But you probably want this to be the username on the remote host, not the one executing the bash script, so:
```suggestion
current_username=\$(whoami)
```

You need to escape the `$()` so that it doesn't get executed by the machine running the `pdsh` command but gets run on the remote machine that `pdsh` is executing the script on.